apps/bare-expo/android/app/build.gradle

```
@@ -75,7 +75,7 @@ import com.android.build.OutputFile
  */
 
 project.ext.react = [
-    enableHermes: false,  // clean and rebuild if changing
+    enableHermes: JS_RUNTIME == "hermes",  // clean and rebuild if changing
```

Maybe something like this:
```suggestion
    enableHermes: findProperty('JS_RUNTIME') ?: 'jsc',  // clean and rebuild if changing
```
Per this guide https://tomgregory.com/gradle-project-properties-best-practices/

---

great suggestion and thanks.
but changed to ` (findProperty('JS_RUNTIME') ?: "hermes") == "hermes"` and proposing hermes by default.
will have `expo-jsc-runtime` module with config plugin for users if they would like to use jsc.